cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,229 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
+`Release 4.1.1 <http://docs.cask.co/cdap/4.1.1/index.html>`__
+=============================================================
+
+New Features
+------------
+
+- :cask-issue:`CDAP-8872` - Changed the Apache Tephra dependency to the 0.11.0-incubating
+  version.
+
+- :cask-issue:`CDAP-9250` - Added support for the Hive version shipped with HDP 2.6.
+
+- :cask-issue:`CDAP-7731` - Added fetch size and transaction flush interval configurations
+  to the Kafka Consumer Flowlet.
+
+- :cask-issue:`CDAP-8902` - Added support for adding, deleting, updating, and retrieving
+  workflow schedules in the CDAP CLI.
+
+- :cask-issue:`CDAP-9117` - Changed the log viewer to only show "user" condensed logs by
+  default. There is an option to "Include System Logs" to show all logs as previously.
+
+- :cask-issue:`CDAP-9141` - Added a 'sparkprogram' plugin type that can be used to run
+  arbitrary Spark code at the beginning or end of a pipeline.
+
+- :cask-issue:`CDAP-9147` - Added pipeline start and finish logs in the condensed view.
+
+- :cask-issue:`CDAP-9235` - Added workspace tabs to the top panel of the Data Preparation (Wrangler) plugin.
+
+- :cask-issue:`CDAP-9281` - Added support for CDH 5.11.0.
+
+- :cask-issue:`CDAP-9327` - Added support in the CDAP UI for the Spark program pipeline
+  plugin.
+
+- :cask-issue:`HYDRATOR-1316` - Added the ability to see logs in preview.
+
+- :cask-issue:`WRANGLER-51` - Added Copy Column to the list of column dropdown directives
+  in the Data Preparation (Wrangler) plugin.
+
+- :cask-issue:`WRANGLER-77` - Added a column-action dropdown menu for Data Preparation
+  directives.
+
+Improvements
+------------
+
+- :cask-issue:`CDAP-7352` - Improved log viewer to always show headers at the top of the
+  screen, even when the user scrolls down.
+
+- :cask-issue:`CDAP-8430` - Modified the Entity list page to show appropriate message when
```

Comment:
appropriate is repeated. Also its the Overview page

``Users can now see an contextual message with appropriate call(s) to action when no entities are found on the Overview page.``